TwitchPlaysAssembly/Src/Commands/GlobalCommands.cs

```diff
@@ -91,6 +91,59 @@ public static void ResetUser([Group(1)] string parameters, string user, bool isW
 		}
 	}
 
+	#region Voting
+	/// <name>Start a vote</name>
+	/// <syntax>vote [action]</syntax>
+	/// <summary>Starts a vote about doing an action</summary>
+	[Command(@"vote (togglevs)")]
+	public static void VoteStart(string user, [Group(1)] bool VSMode) => Votes.StartVote(null, user, VSMode ? "VS Mode Toggle" : "");
+
+	/// <name>Vote</name>
+	/// <syntax>vote [choice]</syntax>
+	/// <summary>Vote with yes or no</summary>
+	[Command(@"vote (yes|voteyea)|(no|votenay)")]
+	public static void Vote(string user, [Group(1)] bool yesVote) => Votes.Vote(user, yesVote);
+
+	/// <name>Remove vote</name>
+	/// <syntax>vote remove</syntax>
+	/// <summary>Removes the vote of a user</summary>
+	[Command(@"vote remove")]
+	public static void RemoveVote(string user) => Votes.RemoveVote(user);
+
+	/// <name>Cancel vote</name>
+	/// <syntax>vote cancel</syntax>
+	/// <summary>Cancels a voting process</summary>
+	/// <restriction>Mod</restriction>
+	[Command(@"vote cancel", AccessLevel.Mod)]
```

Comment:
```suggestion
	[Command(@"vote cancel", AccessLevel.Mod, AccessLevel.Mod)]
```
Unless this is intentional, you need to put the rank twice to also require mod rank in anarchy mode. Same with `vote forceend`.